Tests/src/PublicAPITests/HotspotSequenceTests.cpp

```diff
@@ -220,11 +204,40 @@ CSP_PUBLIC_TEST(CSPEngine, HotspotSequenceTests, CreateHotspotGroupTest)
 	csp::common::Array<csp::common::String> GroupItems {"Hotspot1", "Hotspot2", "Hotspot3"};
 	csp::common::String TestGroupName = "CSP-UNITTEST-SEQUENCE-MAG";
 
+	// Validate sequence creation events.
+	bool CallbackCalled = false;
+	auto* Connection = SystemsManager.GetMultiplayerConnection();
+	Connection->SetHotspotSequenceChangedCallback([&CallbackCalled, &Space, &TestGroupName](const csp::multiplayer::SequenceHotspotChangedParams& Params)
+	{
+		EXPECT_EQ(Params.UpdateType, csp::multiplayer::ESequenceUpdateType::Create);
+		EXPECT_EQ(Params.SpaceId, Space.Id);
+		EXPECT_EQ(Params.HotspotGroupName, TestGroupName);
+		CallbackCalled = true; 
+	});
+
 	csp::systems::HotspotGroup HotspotGroup;
 	CreateHotspotgroup(HotspotSystem, TestGroupName, GroupItems, HotspotGroup);
 
+	WaitForCallback(CallbackCalled);
+	CallbackCalled = false;
+
+	// Validate sequence deletion events.	
+	Connection->SetHotspotSequenceChangedCallback([&CallbackCalled, &Space, &TestGroupName](const csp::multiplayer::SequenceHotspotChangedParams& Params)
+	{
+		EXPECT_EQ(Params.UpdateType, csp::multiplayer::ESequenceUpdateType::Delete);
+		EXPECT_EQ(Params.SpaceId, Space.Id);
+		EXPECT_EQ(Params.HotspotGroupName, TestGroupName);
+		CallbackCalled = true;
+	});
+
 	// Delete sequence
 	DeleteHotspotGroup(HotspotSystem, TestGroupName);
+
+	// Clear out the callback as we have validated what we came here for.
+	WaitForCallback(CallbackCalled);
```

Comment:
What if we get more callbacks than expected?

Reply:
It's a valid question, but given that this approach was taken to conform with the general approach followed currently in CSP tests, I'd prefer to avoid addressing it in this change and save it for one more targeted towards upgrading this pattern.